benchmarks/benchmarks.py

```diff
@@ -105,9 +116,8 @@ def setup(self):
         os.unlink(self.tempfile.name)
         self.tracker = Tracker(self.tempfile.name)
         with self.tracker:
-            for _ in range(MAX_ITERS):
-                self.allocator.valloc(1234)
-                self.allocator.free()
+            self.allocator.valloc(1234)
+            self.allocator.free()
```

Comment:
Looks like the loop here was accidentally dropped.
```suggestion
            for _ in range(LOOPS):
                self.allocator.valloc(1234)
                self.allocator.free()
```